service/src/main/java/com/ge/predix/acs/service/policy/evaluation/PolicyEvaluationServiceImpl.java

```diff
@@ -57,6 +57,7 @@
 import com.ge.predix.acs.service.policy.validation.PolicySetValidator;
 import com.ge.predix.acs.zone.management.dao.ZoneEntity;
 import com.ge.predix.acs.zone.resolver.ZoneResolver;
+import com.thinkaurelius.titan.core.QueryException;
```

Comment:
This violates abstraction - this class should not know/depend on titan classes.  GraphGenericRepository#getEntityWithInheritedAttributes should throw a new exception type (in its package), say  AttributeTraversalLimitException which can be handled in this class.